recipes/mothur/patch

```diff
@@ -0,0 +1,22 @@
+--- Makefile	2018-06-19 16:16:02.000000000 +0200
++++ Makefile	2018-07-19 12:40:55.822132011 +0200
+@@ -22,8 +22,8 @@
+ USEREADLINE ?= yes
+ USEBOOST ?= yes
+ LOGFILE_NAME ?= yes
+-BOOST_LIBRARY_DIR="\"Enter_your_boost_library_path_here\""
+-BOOST_INCLUDE_DIR="\"Enter_your_boost_include_path_here\""
++BOOST_LIBRARY_DIR=${PREFIX}/lib
```

Comment:
It seems that this does not work. PREFIX evaluates to "". Better to set this in the build.sh script.